README.md

```diff
@@ -8,37 +8,892 @@ TASKANA - The open source task management library
 [![Maven Central](https://maven-badges.herokuapp.com/maven-central/pro.taskana/taskana-core/badge.svg)](https://maven-badges.herokuapp.com/maven-central/pro.taskana/taskana-core)
 [![License](http://img.shields.io/:license-apache-blue.svg)](http://www.apache.org/licenses/LICENSE-2.0.html)
 
-TASKANA is a task management component open source library. It can be embedded into your application or be operated standalone if appropriate. Beside the basic task management functionalities, TASKANA adds workbaskets and classifications to control and monitor a large amount of tasks within a larger organization. 
+TASKANA is a task management component open source library. It can be embedded into your application
+or be operated standalone if appropriate. Beside the basic task management functionalities, 
+TASKANA adds workbaskets and classifications to control and monitor a large amount of tasks within 
+a larger organization. 
 
 * Web Site: http://taskana.pro/
 * Demo Environment: https://taskana.azurewebsites.net/taskana
 * Getting Started: https://taskana.atlassian.net/wiki/spaces/TAS/pages/228655109/Getting+started
-* Issue Tracker: https://taskana.atlassian.net
+* Issue Tracker: https://github.com/Taskana/taskana/issues
 * Contribution Guildelines: https://taskana.atlassian.net/wiki/spaces/TAS/pages/274202625/Contribution+Guide
 * License: Apache License, Version 2.0  https://www.apache.org/licenses/LICENSE-2.0
 
-Components
-----------
+---
+_We're not aware of all installations of our Open Source project. However, we love_
+* _listening to your feedback,_
+* _discussing possible use cases with you,_
+* _aligning the roadmap to your needs!_
 
-TASKANA splits up into five components:
+📨 _Please [contact](#contact) us if you need consulting support._
 
-#### TASKANA core
+---
+
+# Table of Contents
+
+* ✨ [Overview](#Overview)
+    * [Tasks](#tasks)
+    * [Workbasket](#workbasket)
+    * [Classification](#classification)
+    * [Routing](#routing)
+    * [Prioritization](#prioritization)
+* [Components](#components)
+    * [TASKANA Lib](#taskana-lib)
+    * [TASKANA REST API](#taskana-rest-api)
+    * [TASKANA workplace](#taskana-workplace)
+    * [TASKANA admin](#taskana-admin)
+    * [TASKANA monitor](#taskana-monitor)
+* 🚀 [Getting Started](#getting-started)
+    * [Requirements](#requirements)
+    * [Wrapper Application](#wrapper-application)
+        * [Spring Boot Example](#spring-boot-example)
+        * [EJB Example](#ejb-example)
+* [Customize Behaviour](#customize-behaviour)
+* 📚 [Releases](#releases)
+* 📨 [Contact](#contact)
+
+
+# ✨Overview
+
+## Tasks
+
+Tasks are the main entity of . Each task has its describing attributes like priority and due 
+date.
+
+Furthermore each task has a state.
+
+And a task holds a reference to the system and business object, it is associated with. This is 
+important since  is meant to be a standalone component.
+
+All tasks are placed in a WORKBASKET to control and direct the handling of the tasks.
```

Comment:
Because it is written here the same way

https://taskana.atlassian.net/wiki/spaces/TAS/overview?homepageId=98306

![image](https://github.com/Taskana/taskana/assets/2970074/b0d9eeff-c93f-498e-a183-e96f911970b3)

Most of the things come directly from confluence :)

Reply:
Could I change this in the commit about uppercase and lowercase?

Reply:
yes of course